src/core_modules/capture-core/utils/converters/date/convertIsoToLocalCalendar.js

```diff
@@ -18,7 +18,7 @@ export function convertIsoToLocalCalendar(isoDate: ?string): string {
         return '';
     }
 
-    const momentDate = moment(isoDate);
+    const momentDate = moment(isoDate).locale('en');
```

Comment:
The app crashed when using a locale that outputs non-"english" numerals from the moment.format function. We probably have problems elsewhere too with the format function. Created a separate ticket to investigate and fix the other problems: https://dhis2.atlassian.net/browse/DHIS2-19014.

The dhis2 ui calendar component always displays "english" numerals no matter which locale is used. For now, we will do the same in the app.

Reply:
@JoakimSM  The UI library actually displays numerals according to the selected locale - for example, when using Arabic/Sudan locale `ar-SD` , it will show Arabic numerals (٠‎, ١‎, ٢‎, ٣‎, etc.) rather than English numerals (0, 1, 2, 3, etc.). 
<img width="318" alt="image" src="https://github.com/user-attachments/assets/abd139bf-d2b6-4b1f-8bb8-b818ab2ec3a3" />

Reply:
Thanks @alaa-yahia for making me aware of this (I was testing using some locales that moment used other numerals for (like 'ar'), but the calendar component displayed the "english" ones). I have looked into this a bit and  made a ticket: https://dhis2.atlassian.net/browse/DHIS2-19045 that we will discuss and decide whether we want to implement this at a later point.